packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -1,13 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import { VmSafe, Vm } from "sphinx-forge-std/Vm.sol";
+import { VmSafe, Vm } from "../../lib/forge-std/src/Vm.sol";
```

Comment:
Note how we import forge-std using a relative path instead of using a remapping. This is because any remapping we might define related to this would also be required in the user repo. So, I decided not to use remappings at all in the `contracts` package. 

TODO
See this comment for info on how we enforce this to prevent regressions.